packages/shared/getComponentName.js

```diff
@@ -9,8 +9,13 @@
 
 import type {Fiber} from 'react-reconciler/src/ReactFiber';
 
+import {REACT_CALL_TYPE} from 'shared/ReactSymbols';
+
 function getComponentName(fiber: Fiber): string | null {
   const {type} = fiber;
+  if (type === REACT_CALL_TYPE) {
+    return '<ReactCall>';
```

Comment:
Why didn’t Fragment need this?

Reply:
I guess because we don't have a test that covers them. I'll add one.

Reply:
Actually I guess the debug tool is supposed to skip over these